src/charm.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+
+"""SSSD Operator Charm."""
+
+import logging
+
+import sssd
+from ops.charm import CharmBase
+from ops.main import main
+from ops.model import ActiveStatus, BlockedStatus
+
+logger = logging.getLogger(__name__)
+
+
+class SSSDCharm(CharmBase):
+    """SSSD Charm."""
+
+    def __init__(self, *args):
+        super().__init__(*args)
+        # Standard Charm Events
+        self.framework.observe(self.on.install, self._on_install)
+        self.framework.observe(self.on.start, self._on_start)
+        # Integrations
+        self.framework.observe(
+            self.on.sssd_ldap_relation_changed, self._on_sssd_ldap_relation_changed
+        )
+        # Client Manager
+        self.sssd = sssd
```

Comment:
I do not think you need this anymore provided how we modified the _sssd.py_ module. You should just be able to call out to the manager using the module name directly:

```python3
sssd.install()
sssd.start()
sssd.save_conf()
...
```

Let me know if you already tried this and it didn't work.

Reply:
By the way, you might want to check this in the GLAuth charm as well.

Reply:
I was wondering if I should remove it or not in case I was doing something not in charm "standards" but it seems to work fine so I updated it to use the module directly.